vxquery-server/src/main/resources/conf/local.xml

```diff
@@ -16,6 +16,7 @@
 -->
 <cluster xmlns="cluster">
     <name>local</name>
+    <indexDirectory>/home/menaka/index</indexDirectory>
```

Comment:
Yes. This was also  a problem that I had also because of the XTests. I think this solution will work fine. So, I'll keep it as target/tmp/indexFolder